### ISSUE TITLE
citgm-smoker-nobuild issue on aix

### ISSUE THREAD

**Issue by targos** [nodejs/build#2558]
https://ci.nodejs.org/job/citgm-smoker-nobuild/1054/nodes=aix71-ppc64/console
https://ci.nodejs.org/job/citgm-smoker-nobuild/1058/nodes=aix71-ppc64/console

It errors with:
```
11:44:41 + node -v
11:44:41 terminate called after throwing an instance of 'std::system_error'
11:44:41   what():  Invalid argument
11:44:41 /tmp/jenkins1947871480275286675.sh: line 31: 26607754 IOT/Abort trap          node -v
```

**Comment by targos** [comment]
@nodejs/citgm-admins

**Comment by richardlau** [comment]
I've added the `aix72-ppc64` label to the job so it will run there as well. 

In terms of AIX 7.1 there was a change to build Node.js 15 and later AIX releases on AIX 7.2 instead of AIX 7.1 (https://github.com/nodejs/build/pull/2541) and the binaries built on AIX 7.2 (i.e. the ones on the download server) don't run on AIX 7.1 (https://github.com/nodejs/build/issues/2547#issuecomment-782107370). We've stated the minimum supported version of AIX to be 7.2 for Node.js 12 onwards but we've only recently got additional AIX 7.2 hosts in the CI which is why most of the jobs were running on the AIX 7.1 hosts. 

Perhaps we should revert https://github.com/nodejs/build/pull/2541 so that we no longer test on AIX 7.1 for the releases that we are building on AIX 7.2? That would align with out support statement in [BUILDING.md](https://github.com/nodejs/node/blob/master/BUILDING.md#platform-list) and could mean we could replace the AIX 7.1 hosts (with more AIX 7.2 ones) after Node.js 10 goes End-of-Life and we switch building Node.js 14 and 12 releases to AIX 7.2 (we started with 15 out of caution). cc @nodejs/platform-aix
